tests/TestCase/Error/DebuggerTest.php

```diff
@@ -105,6 +109,7 @@ public function testExcerpt()
         $this->assertTrue(is_array($result));
         $this->assertCount(4, $result);
 
+        $this->skipIf(defined('HHVM_VERSION'), 'HHVM does not highlight php code');
```

Comment:
What kind of network? WAN? To make stuff work locally it would need to be bundled with cake?